core/spec/models/spree/stock/quantifier_spec.rb

```diff
@@ -12,8 +12,6 @@ module Spree
   module Stock
     describe Quantifier, :type => :model do
 
-      before(:all) { Spree::StockLocation.destroy_all } #FIXME leaky database
```

Comment:
For some reason I'm having to add this one back to get specs passing on master, but they passed fine on 2-4 without this.